src/demo/share/java2d/J2DBench/src/j2dbench/tests/text/TextTests.java

```diff
@@ -288,7 +288,7 @@ static String getString(Object key, int len) {
     }
 
 
-    static HashMap strcache = new HashMap(tscripts.length);
+    static HashMap strcache = HashMap.newHashMap(tscripts.length);
```

Comment:
This code maintains compatibility with jdk1.4, not necessary to change it.

Reply:
@mrserb done.